libexec/rc/rc.subr

```diff
@@ -1301,6 +1302,14 @@ run_rc_command()
 			fi
 
 			if [ "${_rc_svcj}" != jailing ]; then
+				# service can redefine all so
+				# check for valid setup target
+				if [ "$rc_arg" = 'start' -o \
+				    "$rc_arg" = 'restart' -o \
+				    "$rc_arg" = 'reload' ]; then
```

Comment:
What would that look like?

Reply:
In the case of caddy it requires dealing with this extended `reload` concept :

    www/caddy/files/caddy.in:reloadssl_cmd="caddy_execute reload --force ${caddy_flags}"

So when we are in this condition here we also check for existence of reloadssl_setup="YES" with a `if ! checkyesno $_setup; then` -- it requires fixing affected ports but only since the port was extended in this particular way.

I think it's fair to assume this only pertains to custom `$_cmd` definitions. For general service handling `start`, `restart` and `reload` do the right thing since they are the unmodified standard commands then and shouldn't be disabled, but it could also be added for consistency.